packages/headless/src/controllers/breadcrumb-manager/headless-breadcrumb-manager.ts

```diff
@@ -173,30 +173,30 @@ export const buildBreadcrumbManager = (
 
 /**
  * Represents a generic breadcrumb type.
+ *
+ * Can either be a `FacetBreadcrumb`, `NumericFacetBreadcrumb`, `DateFacetBreadcrumb`, or `CategoryFacetBreadcrumb`.
  */
 export interface Breadcrumb<T extends BaseFacetValue> {
```

Comment:
I think we should not export the generic interface, but rather export 4 "pre-packaged" types. I would not expect developers to need to import two interfaces and plug them together.

We are already doing this for the `CategoryFacetBreadcrumb`, which has a different shape, so the jsdoc as it is is slightly incorrect.

Suggestion:
```
export type FacetBreadcrumb = Breadcrumb<FacetValue>
export type NumericFacetBreadcrumb = Breadcrumb<NumericFacetValue>;
export type DateFacetBreadcrumb = Breadcrumb<DateFacetValue>;
```

Once we have those, we should not export the generic `Breadcrumb` interface, and update Atomic.

Are you comfortable making these changes Kaitlyn? If not, I can make the adjustments before handing back to you,

Reply:
Yea, your first suggestion seems pretty straightforward, but I'm hesitant to try this one myself. Why don't you make the changes and I can make the accompanying documentation changes after :)

Reply:
I explored a bit, and saw that our atomic component heavily relies on the generic interface at the moment.
The three types above are exported at the bottom of the file, but not used. I will open a separate jira to look into the atomic implementation, and whether it would be simpler to not use the generic,